crates/astria-core/src/execution/v1/mod.rs

```diff
@@ -49,9 +49,17 @@ pub struct GenesisInfo {
     /// The rollup id which is used to identify the rollup txs.
     rollup_id: RollupId,
     /// The Sequencer block height which contains the first block of the rollup.
-    sequencer_genesis_block_height: tendermint::block::Height,
+    sequencer_start_block_height: tendermint::block::Height,
+    /// The Sequencer block height to stop at.
+    sequencer_stop_block_height: tendermint::block::Height,
```

Comment:
Hmm, I think it would still just restart in this case? Josh introduced some logic such that if it receives a stop height of 0, it acts as if there is no stop block height. But in the case where stop block height is not 0 and is less than start block height, the conductor will go to execute and then just restart again, so the functionality should already be tested, but I can add a specific test for this scenario just to be sure if you'd like. In either case, if the stop height is ever below the start, I think something is wrong on the rollup end